src/openrct2/Editor.cpp

```diff
@@ -544,17 +543,16 @@ namespace Editor
             }
         }
 
-        for (sint32 i = 0; i < MAX_PARK_ENTRANCES; i++)
-        {
-            if (gParkEntrances[i].x == MAP_LOCATION_NULL)
+        for (auto &gParkEntrance : gParkEntrances) {
```

Comment:
`gParkEntrance` makes no sense. It's not a global. Call it `parkEntrance`. Same for banners.